src/app/tasks/tasks.component.scss

```diff
@@ -1 +1,45 @@
 /* stylelint-disable */
+:host {
+  text-align: left;
+}
+
+.tasks-list {
+  position: relative;
+}
+
+.tasks-list-container {
+  overflow: auto;
+}
+
+table.mat-table {
+  width: 100%;
+}
+
+th.mat-header-cell {
+  text-transform: uppercase;
+  font-size: 0.6875rem;
+  padding: 0 0.5rem;
+  white-space: nowrap;
+}
+
+td.mat-cell {
+  font-size: 0.75rem;
+  padding: 0 0.5rem;
+}
+
+.mat-form-field {
+  margin-right: 1rem;
+}
+
+.overlay {
+  position: absolute;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 56px;
+  background: rgba(0, 0, 0, 0.15);
```

Comment:
We could use a variable for black color

Reply:
i think we should do this after a design is approved